sdk/keyvault/azure-keyvault-administration/tests/test_access_control.py

```diff
@@ -0,0 +1,95 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import functools
+import os
+import uuid
+
+from azure.keyvault.administration import KeyVaultAccessControlClient
+from devtools_testutils import KeyVaultPreparer, ResourceGroupPreparer
+import pytest
+
+from _shared.test_case import KeyVaultTestCase
+from _shared.preparer import KeyVaultClientPreparer as _KeyVaultClientPreparer
+
+AccessControlClientPreparer = functools.partial(_KeyVaultClientPreparer, KeyVaultAccessControlClient)
+
+
+class AccessControlTests(KeyVaultTestCase):
+    def __init__(self, *args, **kwargs):
+        super(AccessControlTests, self).__init__(*args, **kwargs)
+        if self.is_live:
+            pytest.skip("test infrastructure can't yet create a Key Vault supporting the RBAC API")
+
+    def get_replayable_uuid(self, replay_value):
+        if self.is_live:
+            value = str(uuid.uuid4())
+            self.scrubber.register_name_pair(value, replay_value)
+            return value
+        return replay_value
+
+    def get_service_principal_id(self):
+        replay_value = "service-principal-id"
+        if self.is_live:
+            value = os.environ["AZURE_CLIENT_ID"]
+            self.scrubber.register_name_pair(value, replay_value)
+            return value
+        return replay_value
+
+    @ResourceGroupPreparer(random_name_enabled=True)
+    @KeyVaultPreparer()
+    @AccessControlClientPreparer()
+    def test_list_role_definitions(self, client):
+        definitions = [d for d in client.list_role_definitions("/")]
+        assert len(definitions)
+
+        for definition in definitions:
+            assert "/" in definition.assignable_scopes
+            assert definition.description is not None
+            assert definition.id is not None
+            assert definition.name is not None
+            assert len(definition.permissions)
```

Comment:
is it possible for things like permissions, name etc to assert on the value?

Reply:
Not without prior knowledge of the values, or another API that enables checking our deserialization (I don't see a candidate). Nothing's coming to my mind short of hardcoding what appear to be the current defaults; I'm reluctant to do that. Any ideas?

Reply:
you're right, i think we can leave it for now (i can't think of anything either)